modAionImpl/src/org/aion/zero/impl/AionBlockchainImpl.java

```diff
@@ -527,21 +528,17 @@ public synchronized void compactState() {
         repository.compactState();
     }
 
-    /**
-     * Processes a new block and potentially appends it to the blockchain, thereby changing the
-     * state of the world. Decoupled from wrapper function {@link #tryToConnect(AionBlock)} so we
-     * can feed timestamps manually
-     */
-    ImportResult tryToConnectInternal(final AionBlock block, long currTimeSeconds) {
+    // TEMPORARY: here to support the ConsensusTest
+    public Pair<ImportResult, AionBlockSummary> tryToConnectAndFetchSummary(AionBlock block, long currTimeSeconds) {
```

Comment:
Let's make a note/Issue/Jira to re-visit this once the critical code has been merged in